pkg/util/iptables/iptables.go

```diff
@@ -82,6 +82,7 @@ type Table string
 const (
 	TableNAT    Table = "nat"
 	TableFilter Table = "filter"
+	TableMangle Table = "mangle"
```

Comment:
Why is this part of the same PR?  Seems unrelated?

Reply:
It's used by `NewFakeIPTables` to cache builtin chains.